### PR TITLE
Switch to using print function for improved Python 3 compatibility.

### PR DESCRIPTION
**Pull request by Onager**
Switch to using print function for improved Python 3 compatibility.

### CODE REVIEW

File: dftimewolf/lib/collectors/grr_hosts.py [hunk 1]
```diff
@@ -159,19 +160,19 @@ def _await_flow(self, client, flow_id):
...
+    print('{0:s}: Downloading artifacts'.format(flow_id))
     collected_file_path = self._download_files(client, flow_id)
 
     if collected_file_path:
-      print '{0:s}: Downloaded: {1:s}'.format(flow_id, collected_file_path)
+      print('{0:s}: Downloaded: {1:s}'.format(flow_id, collected_file_path))
 
   def print_status(self, flow):
-    """Print status of flow.
+    """print(status of flow.
```
Comment (tomchop) [inline dftimewolf/lib/collectors/grr_hosts.py]: Docstring got caught in find / replace

File: dftimewolf/lib/collectors/grr_hunt.py [hunk 2]
```diff
@@ -37,17 +38,17 @@ def _create_hunt(self, name, args):
...
     hunt = self.grr_api.CreateHunt(
         flow_name=name, flow_args=args, hunt_runner_args=runner_args)
-    print '{0:s}: Hunt created'.format(hunt.hunt_id)
+    print('{0:s}: Hunt created'.format(hunt.hunt_id))
     self._check_approval_wrapper(hunt, hunt.Start)
     return hunt
 
   def print_status(self):
-    """Print status of hunt."""
+    """print(status of hunt."""
```
Comment (tomchop) [inline dftimewolf/lib/collectors/grr_hunt.py]: Docstring got caught in find / replace

File: dftimewolf/lib/collectors/grr_hosts.py (hunk 1 above)
Comment (Onager) [reply dftimewolf/lib/collectors/grr_hosts.py]: Done

File: dftimewolf/lib/collectors/grr_hunt.py (hunk 2 above)
Comment (Onager) [reply dftimewolf/lib/collectors/grr_hunt.py]: Done

**Pull request merged**